src/components/HtmlInjection.tsx

```diff
@@ -15,6 +15,10 @@ export function HtmlInjection(props: Props) {
 
 function useScripts(html: Props['html'], onError: Props['onError']) {
     const [strippedHtml, scriptsAttributes] = useMemo(() => {
+        if (typeof document === 'undefined') {
+            return [html, []];
+        }
```

Comment:
Do I understand correctly, that the hook will do the full job in client rendering mode, and directly inject HTML without attaching script tags in SSR?

Reply:
Yes, that's right

Reply:
Ok, I see. Thanks!